src/rfquack_radio.h

```diff
@@ -29,89 +29,94 @@
 #include "rfquack.pb.h"
 
 #include <cppQueue.h>
+#include <radio/RFQnRF24.h>
```

Comment:
Include only based on a `#define RFQUACK_RADIO_<name of module>` value (see [config/radio.h](https://github.com/trendmicro/RFQuack/blob/master/src/config/radio.h)).

Note that in the future, we might support multiple modules simultaneously.